src/modelfree/ppo_baseline.py

```diff
@@ -231,21 +295,60 @@ def default_ppo_config():
     _ = locals()  # quieten flake8 unused variable warning
     del _
 
+    # TODO: auto-create out directory
 
 # TODO: use victim_type
 @ppo_baseline_ex.automain
 def ppo_baseline(_run, env, victim, victim_type, out_dir, exp_name, vectorize,
-                 no_normalize, seed, total_timesteps, network, nsteps, load_path):
+                 no_normalize, seed, total_timesteps, network, nsteps, load_path,
+                 rew_shape_anneal_frac, victim_noise_anneal_frac, victim_noise_param,
+                 env_wrapper_type):
     # TODO: some bug with vectorizing goalie
     if env == 'kick-and-defend' and vectorize != 1:
         raise Exception("Kick and Defend doesn't work with vecorization above 1")
 
     out_dir = setup_logger(out_dir, exp_name)
-    env = make_zoo_vec_env(env_name=env, victim=victim, victim_index=0, no_normalize=no_normalize,
-                           seed=seed, out_dir=out_dir, vector=vectorize)
+    scheduler = Scheduler(lr_func=annealer_collection['default_lr'].get_value)
+
+    if rew_shape_anneal_frac is not None:
+        assert 0 <= rew_shape_anneal_frac <= 1
+        rew_shape_func = LinearAnnealer(1, 0, rew_shape_anneal_frac).get_value
+        scheduler.set_rew_shape_func(rew_shape_func)
+
+        def env_wrapper(x):
+            return RewardShapingEnv(x, reward_annealer=scheduler.get_rew_shape_val)
+    else:
```

Comment:
Should this really be an else? Might we want to do e.g. both HumanoidEnvWrapper and RewardShapingEnv?